Updated /apps /clients api doc with SLO config (#4238)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

    * In App Wizard SAML App, no override attributes are available.

* If Single Logout is supported by the application and the `slo` object is provided in the request, the `spCertificate` object must be present.

> * If Participate Single Logout is supported by the application and the `participateSlo` object is provided in the request, the `spCertificate` object must be present.

Reviewer: ```suggestion
* If Participate Single Logout is supported by the application and the `participateSlo` object is provided in the request, you must include the `spCertificate` object.
```

> * When you update an application, if you don't specify `participateSlo` or `slo` or `spCertificate` the existing configuration persists.

Reviewer: ```suggestion
* When you update an application, if you don't specify `participateSlo`, `slo`, or `spCertificate`, the existing configuration persists.
```

| dpop_bound_access_tokens  <ApiLifecycle access="ea" />                                                    | Indicates that the client application uses Demonstrating Proof-of-Possession (DPoP) for token requests. If omitted, the default value is `false`. If `true`, the authorization server rejects token requests from this client that don't contain the DPoP header.  | Boolean  | TRUE       | FALSE    | TRUE      |
| participate_slo <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Determines if the application participates in Single Logout                      | Boolean  | TRUE       | FALSE    | TRUE      |
| frontchannel_logout_uri <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | URL where Okta sends the logout request                            | URL  | TRUE       | FALSE    | TRUE      |

> | frontchannel_logout_session_required <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Allows you to end a specific user session instead of all active user sessions    | Boolean  | TRUE       | FALSE    | TRUE      |

Reviewer 2: For API reference I wonder if it would be better to be more explicit in the description here and just say that this boolean determines whether we send `sid` and `iss` in the logout request per spec? What do you think @anweshdatla-okta @susanharper-okta ?

Reviewer: hmmmm - good point. what about:

Determines whether Okta sends `sid` and `iss` in the logout request.

> | frontchannel_logout_session_required <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Session details allow you to end a specific user session instead of all active user sessions    | Boolean  | TRUE       | FALSE    | TRUE      |

Reviewer: ```suggestion
| frontchannel_logout_session_required <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Allows you to end a specific user session instead of all active user sessions    | Boolean  | TRUE       | FALSE    | TRUE      |
```

* When you create an app using the App Wizard in the UI, and you specify an app logo for the **Application logo** property, that value is stored as the `logo_uri` value and used as the logo on the application's tile for the dashboard as well as the client consent dialog box during the client consent flow. If you add or modify a `logo_uri` value later, that value is used only on the client consent dialog box during the client consent flow.

> * `participate_slo` can only be enabled for application types `web` and `browser`.

Reviewer: ```suggestion
* You can only enable `participate_slo` for application types `web` and `browser`.
```

            "application_type": "native",
            "consent_method": "TRUSTED",
            "issuer_mode": "CUSTOM_URL",

>             "participate_slo": false.

Reviewer: ```suggestion
            "participate_slo": false,
```

}
```

### Participate Single Logout object

Specifies the Participate Single Logout behavior for a Custom SAML application

| Property  | Description                                                                  | Datatype | Nullable |
| --------- | ---------------------------------------------------------------------------- | -------- | -------- |

> | enabled             | Whether the application supports participateSlo                    | Boolean  | FALSE    |

Reviewer: ```suggestion
| enabled             | Indicates whether the application supports `participateSlo`                    | Boolean  | FALSE    |
```

| logoutRequestUrl    | URL where Okta sends the logout request                            | URL      | TRUE     |

> | bindingType         | HTTP binding used to send an `<LogoutRequest>` message to the RP   | `POST` or `REDIRECT`   | TRUE     |

Reviewer 2: ```suggestion
| bindingType         | HTTP binding used to send a `<LogoutRequest>` message to the SP   | `POST` or `REDIRECT`   | TRUE     |
```

> | sessionIndexRequired | Determines whether Okta sends SessionIndex in the logout request  | Boolean   | TRUE     |

Reviewer 2: nit 
```suggestion
| sessionIndexRequired | Determines whether Okta sends `<SessionIndex>` elements in the logout request  | Boolean   | TRUE     |
```

> | sessionIndexRequired | Allows you to end a specific user session instead of all active user sessions | Boolean   | TRUE     |

Reviewer 2: Same for SAML, would it be better to be more explicit and just say this boolean determines whether we send `SessionIndex` elements in the request?

Author: modified description to "Determines whether Okta sends SessionIndex in the logout request"

> | sessionIndexRequired | Session details allow you to end a specific user session instead of all active user sessions | String   | TRUE     |

Reviewer: ```suggestion
| sessionIndexRequired | Allows you to end a specific user session instead of all active user sessions | String   | TRUE     |
```
